### DISCUSSION TITLE
Configurable / Sub -product prices

### DISCUSSION THREAD

**Discussion by m-overlund** [OpenMage/magento-lts#3591] [Questions & Answers]
Importing product prices from vendors for Configurable products is kind of a nightmare to be honest.
Right now i have to import prices for SKU's (subproducts), and then manually go through every configurable product, to update the differences in price for each configurable attribute.

Same goes for Configurable products with multiple configurable attributes - here you cannot always set the wanted price, as pricing is handled by each attribute and not by each SKU/Subproduct.

I know there's been some modules trying to use subproduct pricing instead, more or less succesful and with some caveats.
(and never understood the percentage price increase for subproducts).

Is this some issues OpenMage should focus on fixing?

**Comment by addison74** [comment]
In certain situations, Magento import/export features are limited. Have you tried Magmi?
